docs/data-apps/video-media/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Video and Media Analytics"
+sidebar_position: 5
+sidebar_label: "Video and Media Analytics"
+---
+
+:::caution
+
+This data app is currently in Private Preview and features may change without notice.
+
+:::
+
+While 91% of businesses use video as a marketing tool, few businesses deeply understand how their users interact with this content and what effect this has on business metrics. Collecting vital engagement data plays a significant role in understanding your audience, optimizing your content, and measuring the impact of your video and media marketing efforts.
+
+Our **Video and Media Analytics** app helps you visualize insights on your video, audio and streaming content, based on data generated by our media plugins and [DBT model](docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-media-player-data-model/index.md). It comes with the following dashboards:
+
+1. Overview: Get a high-level view across all your media and video content performance including 'total watch time', 'plays over time', and 'top-performing content'.
+2. Content: Gain deeper insights into your overall and individual media with impressions to plays, overview audience retention, and deep dive into individual content performance.
```

Comment:
This could be a bit more specific - perhaps something like "Dive deeper into individual media items through an impression to plays funnel, audience retention chart and views over time graph"